azurerm/resource_arm_storage_queue_test.go

```diff
@@ -217,8 +245,62 @@ func testAccAzureRMStorageQueue_requiresImport(rInt int, rString string, locatio
 
 resource "azurerm_storage_queue" "import" {
   name                 = "${azurerm_storage_queue.test.name}"
-  resource_group_name  = "${azurerm_storage_queue.test.resource_group_name}"
   storage_account_name = "${azurerm_storage_queue.test.storage_account_name}"
 }
 `, template)
 }
+
+func testAccAzureRMStorageQueue_metaData(rInt int, rString string, location string) string {
+	template := testAccAzureRMStorageQueue_template(rInt, rString, location)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_storage_queue" "test" {
+  name                 = "mysamplequeue-%d"
+  storage_account_name = "${azurerm_storage_account.test.name}"
+
+  metadata = {
+    hello = "world"
```

Comment:
Should we be checking case here:
```suggestion
    HellO = "WorlD!"
```

Reply:
the API's pedantic about keys here:

```
metadata.Rick:        "" => "M0rty"
metadata.hello:       "world" => "world"
metadata.rick:        "M0rty" => ""
```

since this is a common method, I'll add some validation in the next PR to ensure these keys are lower-cased